### PR TITLE
Allow find command to search through all fields

### LINKED ISSUE #123: Add Ability to Search for Events and Vendors by All Fields
Similar to #117 but focused on tags instead. Would help with ease of searching a bit.

### PR DESCRIPTION
**Pull request by kahsuann**
Resolves #123

Event's EventNameContainsKeywordsPredicate -> EventContainsKeywordsPredicate
Vendor's NameContainsKeywordsPredicate -> VendorContainsKeywordsPredicate

### CODE REVIEW

**Review by Jasperabez — APPROVED** [review]
LGTM! did some functionality testing on my own and the test cases seems to be well covered

File: src/main/java/seedu/eventtory/logic/commands/util/PredicatePreviewUtil.java [hunk 1]
```diff
@@ -20,7 +20,7 @@ public class PredicatePreviewUtil {
      * @return a filtered list of vendors
      */
     public static FilteredList<Vendor> getPreviewofFilteredVendors(Model model,
```
Comment (C5hives) [inline src/main/java/seedu/eventtory/logic/commands/util/PredicatePreviewUtil.java:22]: Oops, camelCase issue here that was likely missed. Should be `getPreviewOfFilteredVendors` (with a capital 'O' in 'Of').

File: src/main/java/seedu/eventtory/logic/commands/util/PredicatePreviewUtil.java [hunk 2]
```diff
@@ -35,7 +35,7 @@ public static FilteredList<Vendor> getPreviewofFilteredVendors(Model model,
      * @return a filtered list of events
      */
     public static FilteredList<Event> getPreviewofFilteredEvents(Model model,
```
Comment (C5hives) [inline src/main/java/seedu/eventtory/logic/commands/util/PredicatePreviewUtil.java:37]: Same as previous.

**Review by C5hives — CHANGES_REQUESTED** [review]
Just one naming issue that was missed previously!

**Review by C5hives — APPROVED** [review]

**Pull request merged**